JarrydWorland/2021_Capstone_GP2_Team
# Enemy zapper

## Description
This PR adds the new zapper enemy and relevant assets.

## How to test
Currently the enemy is not placed in any rooms, so place it yourself and ensure the enemy attacks and takes damage correctly. It should be noted that **If you place the enemy outside the hierarchy of a room it will follow you out-of-bounds. This is expected.**

If you wish to avoid this, the most consistent way to test is to place the enemy within some of the room prefabs so that they spawn naturally, this will ensure they are part of the room they spawn in.
